security/passwords.rst

```diff
@@ -626,7 +626,7 @@ you must register a service for it in order to use it as a named hasher:
         security:
             # ...
             password_hashers:
-                app_hasher:
+                App\Entity\User:
```

Comment:
We'd need to also update the XML and PHP examples.

And, at the end of this section we have this, that should also be updated:

> This creates a hasher named ``app_hasher`` from a service [...]

Reply:
fixed